src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="home-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--start-day-sun calendar--month-length-28">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day--sun calendar--month-length--28">
```
![image](https://user-images.githubusercontent.com/96409563/234958108-12415a4d-1150-4d53-a4a2-c7e986218565.png)